media/js/base/mozilla-image-helper.js

```diff
@@ -38,8 +34,7 @@ Mozilla.ImageHelper.initPlatformImages = function() {
     $('.platform-img').each(function() {
         var $img = $(this);
         var data_attribute = 'src-';
-        var is_high_res = $img.data('high-res') && Mozilla.ImageHelper.isHighDpi();
-        var suffix;
+        var is_high_res = $img.data('high-res');
```

Comment:
Looking at the HTML that is now generated for images using the `high_res_img` helper, there seems to no longer be a `high-res` attribute set on the element. So, I guess we do not need this?